integration_test/test_compute_metrics.py

```diff
@@ -0,0 +1,89 @@
+from sigopt.xgboost.compute_metrics import (
+  compute_classification_report,
+  compute_mae,
+  compute_mse,
+  compute_accuracy
+)
+from sklearn.metrics import (
+  accuracy_score,
+  classification_report,
+  mean_absolute_error,
+  mean_squared_error
+)
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+import xgboost as xgb
+import numpy as np
+
+
+
+
+def assert_classification_metrics(y_true, y_pred):
+  report_compute = compute_classification_report(y_true, y_pred)
+  report_sklearn = classification_report(y_true, y_pred, output_dict=True, zero_division=0)
+
+  # Check classification metrics
+  assert np.abs(report_compute['weighted avg']['precision'] - report_sklearn['weighted avg']['precision']) < 1e-8
+  assert np.abs(report_compute['weighted avg']['recall'] - report_sklearn['weighted avg']['recall']) < 1e-8
+  assert np.abs(report_compute['weighted avg']['f1-score'] - report_sklearn['weighted avg']['f1-score']) < 1e-8
+  assert np.abs(compute_accuracy(y_true, y_pred) - accuracy_score(y_true, y_pred)) < 1e-8
+
+
+def assert_regression_metrics(y_true, y_pred):
+  assert np.abs(mean_absolute_error(y_true, y_pred) - compute_mae(y_true, y_pred)) < 1e-8
+  assert np.abs(mean_squared_error(y_true, y_pred) - compute_mse(y_true, y_pred)) < 1e-8
+
+
+class TestComputeMetrics(object):
+  def test_classification_metrics_xgb(self):
+    iris = datasets.load_iris()
+    X = iris.data
+    y = iris.target
+    num_class = 3
+    X_train, X_test, Y_train, Y_test = train_test_split(X, y, test_size=0.3)
+    D_train = xgb.DMatrix(X_train, label=Y_train)
+    D_test = xgb.DMatrix(X_test, label=Y_test)
+
+    params = {
+      'eta': 0.3,
+      'num_class': num_class,
+      'objective': 'multi:softmax'
+    }
+    bst = xgb.train(params, D_train)
+    preds = bst.predict(D_test)
+    xgb_preds = np.round(preds)
+    xgb_true = Y_test
+    assert_classification_metrics(xgb_true, xgb_preds)
```

Comment:
I think this portion can live in integration_test.

Reply:
Actually can I just delete this test? Doesn't seem necessary tbh

Reply:
That way we can just have all the compute_metrics tests in one place

Reply:
Yea, you can. You will still need to add some tests to the integration_test so that we know we can verify `run.get_metric` are returning the appropriate values.